server/src/main/java/org/opensearch/action/admin/indices/rollover/MetadataRolloverService.java

```diff
@@ -309,20 +309,46 @@ static List<AliasAction> rolloverAliasToNewIndex(
         String oldIndex,
         String newIndex,
         boolean explicitWriteIndex,
-        @Nullable Boolean isHidden,
+        AliasMetadata aliasMetada,
```

Comment:
Typo here on "aliasMetada", should be "aliasMetadata"

Reply:
@andrross good catch! Tnx!